### PR TITLE
Save a grouped component as a single glyph

### PR DESCRIPTION
**Pull request by nminnie**
@vigliensoni 
I have fixed the logic error behind saving grouped components.
A grouped component now should be saved as only one glyph.
Please test it and let me know if you encounter any issues!

### CODE REVIEW

File: frontend/public/js/app/App.js [hunk 1]
```diff
@@ -253,8 +253,11 @@ var App = Marionette.Application.extend(
          */
         finalizeAndQuit: function ()
         {
+            // Assume default parameters for grouping before finalizing 
+            var userSelections = {'distance': '4', 'parts': '4', 'criterion': 'min', 'func': 'Bounding Box', 'graph': '16'}
```
Comment (vigliensoni) [inline frontend/public/js/app/App.js]: Shouldn't these values be asked before finalization? Or, at least, use the default values used in "Group glyphs and reclassify."

File: rodan_job/base.py [hunk 2]
```diff
@@ -527,7 +527,7 @@ def run_my_task(self, inputs, settings, outputs):
 
         # Automatic grouping and reclassifying
         elif settings['@state'] == ClassifierStateEnum.GROUP_AND_CLASSIFY:
-            # CLASSIFYING STAGE
+            # GROUP AND CLASSIFY STAGE
```
Comment (vigliensoni) [inline rodan_job/base.py]: Correct. If we are grouping and classifying we should ask the user how to perform the grouping. 

I think you can get some ideas about what these values mean by checking https://github.com/DDMAL/Interactive-Classifier/wiki/How-to-Use#grouping-function or, better, by reading the Gamera code.

**Comment by vigliensoni** [comment]
Excellent.

**Comment by nminnie** [comment]
@vigliensoni 
This now works both with or without training data, so there is no classification upon 'Finalize and save'.
The GameraXML output will reflect what the user sees in the browser.

**Pull request merged**